tests/subscriptions/__init__.py

```diff
@@ -174,7 +174,10 @@ def setup_teardown(self, clickhouse_db: None) -> None:
             gen_span_message(self.base_time + timedelta(minutes=tick))
             for tick in range(self.minutes)
         ]
-        write_raw_unprocessed_events(spans_storage, messages)
+        extra_messages = [
+            gen_span_message(self.base_time - timedelta(hours=4)) for _ in range(2)
+        ]
```

Comment:
adding a couple extra messages outside the time bounds of the subscription execution so I know those don't get included in the count